cmdline/bin/dcat.dart

```diff
@@ -36,34 +36,35 @@ void main(List<String> arguments) {
   dcat(paths, argResults[LINE_NUMBER]);
 }
 
-Future dcat(List<String> paths, bool showLineNumbers) {
+Future dcat(List<String> paths, bool showLineNumbers) async {
```

Comment:
I'm not sure what you mean. (Btw, this isn't ready for review yet. It's just up there so people like shams can see what I've done, so far.)